app/src/main/java/com/example/playlistmaker/media/ui/MediaFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.playlistmaker.media.ui
+
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.playlistmaker.R
+import com.example.playlistmaker.media.ui.adapters.MediaPagerAdapter
+import com.example.playlistmaker.media.ui.viewmodels.FavoriteTracksViewModel
+import com.example.playlistmaker.media.ui.viewmodels.PlaylistsViewModel
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class MediaFragment : Fragment(R.layout.activity_media) {
+
+    private lateinit var viewPager: ViewPager2
+    private lateinit var tabLayout: TabLayout
+    private lateinit var title: TextView
+
+    private val favoriteTracksViewModel: FavoriteTracksViewModel by viewModel()
+    private val playlistsViewModel: PlaylistsViewModel by viewModel()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewPager = view.findViewById(R.id.viewPager)
+        tabLayout = view.findViewById(R.id.tabLayout)
+        title = view.findViewById(R.id.title)
+
+        title.text = getString(R.string.media)
+
+        val pagerAdapter = MediaPagerAdapter(requireActivity())
+        viewPager.adapter = pagerAdapter
+
+        TabLayoutMediator(tabLayout, viewPager) { tab, position ->
+            tab.text = when (position) {
+                0 -> getString(R.string.favorite_tracks)
+                1 -> getString(R.string.Playlists)
+                else -> ""
```

Comment:
Здесь тоже можно упростить:
```suggestion
                else -> getString(R.string.Playlists)
```
Плюс айди строки лучше с маленькой буквы называть